dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+1. Then the AdUnit is processed:
+    1. The first bid requires that the label "display" be present in the array. It's not, so that bid is skipped.
+    1. The second bid requires that either "phone" or "tablet" be present. Since tablet is in the label array, that bid is activated and the correct placement is sent to bidderA.
+
+## Use Case: What if some AdUnit auctions should be skipped entirely for some devices?
+
+Say there's a responsive page where one of the AdUnits only supports larger sizes, so it doesn't make sense
+on phones. We really don't want to use that AdUnit for mobile users. Wouldn't it be great if `labelAny` and `labelAll` could be applied to the entire AdUnit? They can. Here's an example:
+
+{% highlight js %}
+
+var AdUnits = [{
+    code: "ad-slot-1",
+    sizes: [[768,90]], 
+    labelAny: ["display", "tablet"], // skip the ad unit entirely for phones
+    bids: [
+        {
+            bidder: "bidderA",  // no conditions
+            params: {
+                placement: "1111"
+            }
+       },{
+            bidder: "tabletBidder",
+            labelAny: ["tablet"], // additional conditions can be applied to bids
+            params: {
+                placement: "2222"
+            }
+       }
+   ]
+}]
+
+{% endhighlight %}
+
+## Use Case: What if some bid requests apply only to users originating certain countries? 
+
+Labels aren't constrained to describing device size -- they can be used for any condition the page wants
+to define. To support this, labels can now be passed into the `requestBids()` function as an argument.
+
+A specific use case: suppose that a certain bidder doesn't have a data center outside of a
+certain region. It's really not worth sending them bid
+requests for users outside of their geographic area. Assuming the page can figure out where the user's from,
+a label can be implemented and applied to make the bid conditional.
+
+{% highlight js %}
+// page logic determines the 'europeanUser' boolean
+If (europeanUser) {
+    reqArgs={labels:['eur']};
+} 
+requestBids(reqArgs);
+{% endhighlight %}
+
+Then this label can be applied to conditions in the AdUnit just like labels that originate from sizeConfig. E.g.
+
+{% highlight js %}
+var AdUnits = [{
+    code: "ad-slot-1",
+    sizes: [[768,90], [468,60], [320,50]], 
+    bids: [
+       {
+            bidder: "euroMobileBidder",
+            labelAll: ["eur", "phone"],
+            params: {
+                placement: "9876"
+            }
+       },
+       ...
+   ]
+}]
+{% endhighlight %}
+
+This example shows that the 'euroMobileBidder' is only interested in receiving bids that have **both**
+labels: both "eur" as passed into requestBids and "phone" as created by sizeConfig.
+
+
+## Syntax Notes
+
+Probably you've gotten the gist of the feature by now, but it's still worth laying out the syntax a little more. In Prebid 1.0, label targeting is constrained to two relatively simple operators: `labelAny` and `labelAll`.
```

Comment:
I strongly believe this information should be added to [the `labels` section of the API reference](http://prebid.org/dev-docs/publisher-api-reference.html#sizeConfig-Labels) since it is reference information (not "how to" like the rest of this page).

In other words, the API reference page should contain "how stuff works", with this page being "how to use it for this use case".
Random notes on text:

First sentence makes assumptions about the reader that may not be true.

We should not refer to "Prebid 1.0" since it's timely and will need to be edited out relatively soon.

This section should be called "Label Targeting" IMO, and it could start from where you write "Label targeting is constrained to "

Also we should not describe them as "relatively simple", that's a value judgment IMO.  We can try to make the examples simple and clear though!

An important question I have is: how do you debug why your labels logic isn't matching/firing?  If you have some docs or know-how on that that you could contribute (which IMO should also go in the API reference) that would be awesome.